cmd/dagger/project.go

```diff
@@ -127,6 +138,12 @@ var projectInitCmd = &cobra.Command{
 		if err := os.WriteFile(proj.local.path, cfgBytes, 0644); err != nil {
 			return fmt.Errorf("failed to write project config: %w", err)
 		}
+		if template != "" && !noTemplate {
+			// nolint:gosec
+			if err := os.WriteFile(filepath.Join(filepath.Dir(proj.local.path), templateFileName), []byte(template), 0644); err != nil {
```

Comment:
Note to self: this should error out if it's going to overwrite an existing `main.go`.